server/interview/views.py

```diff
@@ -243,25 +309,29 @@ def post(self, request):
                     for interview in paired_interviews
                 ],
                 "unpaired_members": unpaired_members,
+                "failed_paired_emails": failed_paired_emails,
+                "failed_unpaired_emails": failed_unpaired_emails,
             },
             status=status.HTTP_201_CREATED,
         )
 
+    def get(self, request):
+        logger.debug("GET request received for PairInterview")
+        return Response({"detail": "This endpoint is for pairing interviews. Use POST to pair interviews."})
 
-class NotifyInterview(APIView):
-    # TODO: Change permission class to isAdminUser or using a key service
-    permission_classes = [IsAuthenticated]
+# class NotifyInterview(APIView):
+#     permission_classes = [IsAdminUser]
 
-    def post(self, request):
-        logger.debug("POST request received for NotifyInterview")
-        # TODO: Implement the notification service
-        logger.info("Notification sent (placeholder)")
-        return Response({"detail": "Notification sent."})
+#     def post(self, request):
+#         logger.debug("POST request received for NotifyInterview")
+#         # TODO: Implement the notification service
+#         logger.info("Notification sent (placeholder)")
+#         return Response({"detail": "Notification sent."})
 
 
 class InterviewQuestions(APIView):
     # TODO: Change permission class to isAdminUser
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAdminUser]
```

Comment:
This is the wrong permission, needs to be `IsAdmin`. This is the import: `from custom_auth.permissions import IsAdmin`

Reply:
Just checking, is the `IsAdminUser` in `questions/views.py` intentional?

Reply:
No, that is changed in my PR I'm waiting for you to review :P